site/docs/components/dialog/examples.mdx

```diff
@@ -57,21 +57,27 @@ Dialog sizes are typically used for different scenarios:
 For default dialogs, you can pass the `accent` prop to `DialogHeader` which accepts a boolean to render an accent bar in the header of the dialog. You cannot render the accent bar when a status is also being rendered.
 
 </LivePreview>
+<LivePreview componentName="dialog" exampleName="WithHeader" >
+  ##  Header
 
-<LivePreview componentName="dialog" exampleName="CloseButton" >
-## Close button
-
-You can also render the `DialogCloseButton` as a child of the `Dialog` component to render a close button in the top right corner of the dialog. You can pass the `onClick` prop like with any `Button` component to handle the close button click.
+`DialogHeader` follows our standardized header for container components and app regions, and it can be added to provide a structured header for dialog. The header includes a title and actions that follows our [Header Block pattern](/salt/patterns/header-block).
 
 </LivePreview>
-
 <LivePreview componentName="dialog" exampleName="Preheader" >
   ## Preheader
 
 You can use the optional `preheader` prop to render a preheader above the main header of your dialog
 
 </LivePreview>
 
+<LivePreview componentName="dialog" exampleName="CloseButton" >
+## Close button
```

Comment:
If this is solely for `DialogCloseButton`, maybe add "deprecated" to the heading text as well. like button variants, add move to bottom of examples